R/merge_hydrofabrics.R

```diff
@@ -51,12 +52,31 @@ build_new_id_table = function(x,
     st_drop_geometry() %>%
     renamer()
   
-  if("set" %in% names(fl)) {
-    fl <- select(fl, oldID = id, set)
-  } else {
-    fl = select(fl, oldID = id, member_comid)
-  }
+  # need to get member_comid for each flowpath out of the lookup table
```

Comment:
As we discussed in chat -- if the member_comid isn't already available, we have to go grab it from the lookup table.